app/scripts/controllers/organization/AddHolController.js

```diff
@@ -117,6 +117,35 @@
                     location.path('/holidays');
                 });
             };
+            scope.submit=function(){
+			  var fromtxt=angular.element('#fromDate').val();
+			  var toDatetxt=angular.element('#toDate').val();
+			  var repaymentsRescheduleTotxt=angular.element('#repaymentsRescheduledTo').val();
+			  if(new Date(fromtxt)<new Date()||new Date(toDatetxt)<new Date()||new Date(repaymentsRescheduleTotxt)<new Date()){
```

Comment:
Use dateFilter(model, scope.df) to read instead of doing a val() on an angular element and casting it to date as this does not take into account that the date format can be changed from user settings in which case the cast to date will fail or even worse, a different date will be validated instead of intended date.